cmplxs/cmplxs_test.go

```diff
@@ -725,6 +725,62 @@ func TestMul(t *testing.T) {
 	}
 }
 
+func TestMulConj(t *testing.T) {
+	s1 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	s2 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	ans := []complex128{2, 8, 18}
```

Comment:
Why not "want"?

Reply:
This reflects the style that exists in the rest of the file which came from floats.

I'll file an issue to go over these since there were a lot picked up and they will all exist in floats as well.